src/validator/CreditCardValidator.java

```diff
@@ -0,0 +1,62 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardValidator implements CreditCardChecking {
+
+	@Override
+	public boolean checkValidity(CreditCardVendor ccv, String number) {
```

Comment:
checkValidity name can be simplify to validate.

Reply:
String number is basically a card number. So it should be named cardNumber rather than number.